mods/page_save.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+
+/**
+ * page_save module
+ *
+ * Sends the wikitext of a page to the MW API for saving
```

Comment:
> With the current split I fear that it might become a bit hard to figure out which module handles what.

That's true. That makes me think that perhaps we should have a different split - the parsoid module only deals with interfacing Parsoid, while a 'page_content' module handles retrieval / save of pages, just like we did with the page_revision module - it handles the logic of it all while handing off the necessary calls to the action module.